src/pages/home/report/ReportActionsView.js

```diff
@@ -145,6 +145,7 @@ class ReportActionsView extends React.Component {
             this.updateNewMarkerAndMarkReadOnce();
         }
 
+        Report.openReport(this.props.reportID);
```

Comment:
I would think that we would remove the call to `updateNewMarkerAndMarkReadOnce()` no? Because `OpenReport` should update the last read since we'll no longer need to wait for the report data to load.

Reply:
Yeah that sounds correct, but we are still not ready to include this yet because this logic is already handled by the call to `updateNewMarkerAndMarkReadOnce()`